src/components/shared/LocationDescription.tsx

```diff
@@ -4,18 +4,19 @@ import useTrackLinkContext from "decentraland-gatsby/dist/context/Track/useTrack
 import useFormatMessage from "decentraland-gatsby/dist/hooks/useFormatMessage"
 import TokenList from "decentraland-gatsby/dist/utils/dom/TokenList"
 
-import { AggregatePlaceAttributes } from "../../../entities/Place/types"
-import { explorerPlaceUrl } from "../../../entities/Place/utils"
-import { SegmentPlace } from "../../../modules/segment"
-import DislikeBox from "../../Button/DislikeBox"
-import FavoriteBox from "../../Button/FavoriteBox"
-import JumpInPositionButton from "../../Button/JumpInPositionButton"
-import LikeBox from "../../Button/LikeBox"
-import ShareBox from "../../Button/ShareBox"
+import { AggregatePlaceAttributes } from "../../entities/Place/types"
+import { explorerPlaceUrl } from "../../entities/Place/utils"
+import { SegmentPlace } from "../../modules/segment"
+import FavoriteBox from "../Button/FavoriteBox"
+import JumpInPositionButton from "../Button/JumpInPositionButton"
+import ShareBox from "../Button/ShareBox"
+import WorldLabel from "../World/WorldLabel/WorldLabel"
+import { Likes } from "./Likes"
 
-import "./PlaceDescription.css"
+import "./LocationDescription.css"
 
 export type PlaceDescriptionProps = {
```

Comment:
I think that if we change the file name from `PlaceDescription` to `LocationDescription`, we should also change the name of the properties, the name of the function (and maybe the names of all the classes), to make a cleaner job.